lib/scarpe/para.rb

```diff
@@ -1,13 +1,31 @@
 module Scarpe
   class Para
-    def initialize(app, text)
+    def initialize(app, text, options = {})
```

Comment:
I think we should consider if we want to use explicit kw args or options hash in general

Reply:
I like that the internal app sits in the middle and deals with all the splatting and keyword args. It feels nice to be dealing with options at this level. But I don't have a strong opinion on it.

Reply:
Since there's so many html attrs that could be added I think having a catch all is a good thing. I would venture to say it may be helpful to have explicit kwargs for the options we know we need to apply transformations to.

i.e. Someone wants to add a title to a para (for whatever reason) we know we dont need to transform that so let the options "catch all" handle it. But for kwargs that aren't html attrs (like stroke) or ones we know we need to transform (like style) we can make explicit kwargs for.

Reply:
I think I understand. I'll have a crack and see how it goes. Thanks for the input. :+1:

Reply:
@DRBragg @TobiasBales LMK if this looks any more sensible now. 🤞

Reply:
@DRBragg makes sense to me and @christhesoul sorry for not replying yesterday, glad you merged it and works for me